data/json/vitrification/vitrified_farm.json

```
@@ -0,0 +1,490 @@
+    "nested_mapgen_id": "diningroom_6x6_vitrified",
+    "object": {
+      "mapgensize": [ 6, 6 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "      ",
+        " BLB a",
+        " BLB I",
+        " BLB I",
+        " BLB y",
+        "      "
+      ],
+      "palettes": [ "house_vitrified_nest_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "a nested map for lawns",
+    "nested_mapgen_id": "firepit_5x5_vitrified",
+    "object": {
+      "mapgensize": [ 5, 5 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        " ccc ",
+        " f   ",
+        "  i  ",
+        "   f ",
+        " ccc "
+      ],
+      "palettes": [ "vitrified_garden_palette" ],
+      "place_items": [ { "item": "stash_wood", "x": 1, "y": 2, "chance": 100, "repeat": [ 2, 10 ] } ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "a nested map for lawns",
+    "nested_mapgen_id": "greenhouse_6x6_vitrified",
+    "object": {
+      "mapgensize": [ 6, 6 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "  s   ",
+        "BBCBB ",
+        "D4R4Dc",
+        "B5R2Bc",
+        "D2R3Dc",
+        "BBCBB "
+      ],
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "palettes": [ "vitrified_garden_palette" ],
+      "terrain": { "s": "t_vitrified_dirt" },
+      "place_item": [ { "item": "glass_apple", "x": 1, "y": 3, "chance": 100 } ]
+    }
+  },
+  {
+    "//": "Home office. Cardinal direction indicates which wall the desk is on.",
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "home_office_4x4_vitrified",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "Prrr",
+        " Yh ",
+        "    ",
+        " RHH"
+      ],
+      "palettes": [ "house_vitrified_nest_palette" ],
+      "furniture": {
+        "P": "f_vitrified_bookcase",
+        "R": "f_vitrified_bookcase",
+        "r": "f_vitrified_desk",
+        "H": "f_vitrified_sofa",
+        "h": "f_vitrified_chair"
+      },
+      "terrain": {
+        "P": "t_vitrified_floor",
+        "R": "t_vitrified_floor",
+        "r": "t_vitrified_floor",
+        "H": "t_vitrified_floor",
+        "h": "t_vitrified_floor"
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "nested_mapgen_id": "bedroom_4x4_child_vitrified",
+    "//": "compass direction indicates head of bed",
+    "object": {
+      "mapgensize": [ 4, 4 ],
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "QaO ",
+        "    ",
+        " EE ",
+        "LEEL"
+      ],
+      "palettes": [ "house_vitrified_nest_palette" ],
+      "terrain": { "Q": "t_vitrified_floor" },
+      "furniture": { "Q": "f_vitrified_bookcase_talker" }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "compass direction indicates head of bed",
+    "nested_mapgen_id": "bedroom_5x4_adult_vitrified",
```

```suggestion
    "method": "json",
    "nested_mapgen_id": "bedroom_5x4_adult_vitrified",
```
Again I get that it doesn't fill the 5x5 but I'm pretty sure that's the naming that's mostly used